contracts/src/v0.8/dev/ocr2/OCR2Base.sol

```diff
@@ -253,13 +253,32 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
 
   function _afterSetConfig(uint8 _f, bytes memory _onchainConfig) internal virtual;
 
-  function _report(
+  /**
+   * @dev hook to allow additional validation of the report by the extending contract
+   * @param configDigest separation tag for current config (see configDigestFromConfigData)
+   * @param epochAndRound 27 byte padding, 4-byte epoch and 1-byte round
+   * @param report serialized report
+   */
+  function _validateReport(
```

Comment:
Do we expect the extending contract to revert if the report is not valid?

Perhaps we can alter this to return a boolean, then call it like this `require(_validateReport(...))`

Reply:
Yeah that's exactly how I was thinking.
Adding this.